controllers/replication.storage/pvc.go

```diff
@@ -56,3 +58,37 @@ func (r VolumeReplicationReconciler) getPVCDataSource(logger logr.Logger, req ty
 
 	return pvc, pv, nil
 }
+
+// annotatePVCWithOwner will add the VolumeReplication details to the PVC annotations.
+func (r *VolumeReplicationReconciler) annotatePVCWithOwner(ctx context.Context, logger logr.Logger, req types.NamespacedName, pvc *corev1.PersistentVolumeClaim) error {
+	if pvc.ObjectMeta.Annotations == nil {
+		pvc.ObjectMeta.Annotations = map[string]string{}
+	}
+
+	ownerName := pvc.ObjectMeta.Annotations[replicationv1alpha1.VolumeReplicationNameAnnotation]
+	ownerNamespace := pvc.ObjectMeta.Annotations[replicationv1alpha1.VolumeReplicationNamespaceAnnotation]
+
+	if ownerName == "" {
+		pvc.ObjectMeta.Annotations[replicationv1alpha1.VolumeReplicationNameAnnotation] = req.Name
+		pvc.ObjectMeta.Annotations[replicationv1alpha1.VolumeReplicationNamespaceAnnotation] = req.Namespace
+
+		err := r.Update(ctx, pvc)
+		if err != nil {
+			logger.Error(err, "Failed to update PVC annotation", "Name", pvc.Name)
+
+			return fmt.Errorf("failed to update PVC %s annotation for VolumeReplication: %w",
+				pvc.Name, err)
+		}
+
+		return nil
+	}
+
+	if ownerName != req.Name || ownerNamespace != req.Namespace {
+		logger.Info("cannot change the owner of PVC", "PVC name", pvc.Name, "current owner", ownerName, "requested owner", ownerName)
```

Comment:
```suggestion
		logger.Info("cannot change the owner of PVC",
		"PVC name", pvc.Name,
		"current owner", fmt.Sprintf("%s/%s",ownerNamesapce,ownerName),
		"requested owner", fmt.Sprintf("%s/%s",req.Namesapce,req.Name))
```

requested owner pointed to ownerName again at the end 

and add namespace here too?

Reply:
a namespace is not required removed it. A namespace is not required because the VR and PVC should be on the same namespace